package.json

```diff
@@ -43,6 +44,13 @@
     "type-check": "tsc --noEmit --pretty",
     "watch": "gulp watch"
   },
+  "resolutions": {
+    "graphql": "^0.12.3",
+    "babel-plugin-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/babel-plugin-relay-1.5.0-plugin.2.tgz",
+    "relay-compiler": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/relay-compiler-1.5.0-plugin.2.tgz",
+    "relay-runtime": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/relay-runtime-1.5.0-plugin.2.tgz",
+    "react-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/react-relay-1.5.0-plugin.2.tgz"
+  },
```

Comment:
I figured it would be clearest to explicitly resolve all of our patches here similar to the [most recent example](https://github.com/alloy/relay-examples/tree/master/todo), and then update / remove when ready.